features/AuthController.feature

```diff
@@ -56,6 +82,66 @@ Feature: Auth Controller
       | "_STRING_" | "test-admin"   | -1          | -1                | "Sigfox API key"    | "_STRING_"          |
 
+  # auth:login =================================================================
+
+  @security @http
+  Scenario: Auth login with cookies
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
+      | cookieAuth | true                                                 |
+    Then The raw response should match:
+      | headers.set-cookie | [ /authToken=[^;]+;.*/ ] |
+    Then I should receive a result matching:
+      | _id       | "_STRING_"     |
+      | expiresAt | "_NUMBER_"     |
+      | ttl       | "_NUMBER_"     |
+      | jwt       |  "_UNDEFINED_" |
+  
+  # auth:logout ================================================================
+
+  @security @http
+  Scenario: Auth logout with cookies
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
+      | cookieAuth | true                                                 |
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                                      |
+      | action     | "logout"                                                    |
+      | cookieAuth | true                                                        |
+      | headers    | { cookie: this.props.rawResponse.headers['set-cookie'][0] } |
+    Then The raw response should match:
+      | headers.set-cookie | [ /authToken=null;.*/ ] |
+    And I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
+      | cookieAuth | true                                                 |
+
+  # auth:refreshToken ==========================================================
+
+  @security @http
+  Scenario: Auth refreshToken with cookies
+    Given I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
```

Comment:
```suggestion
      | body.username       | "test-admin"  |
      | body.password        | "password"  |
```

Reply:
Same as above, It seems that I cannot construct "object on the fly"